docs/source/conf.py

```diff
@@ -86,10 +86,9 @@
     "github_repo": "sqlfluff",
     # Github Fork button
     "github_banner": True,
-    # Github link button
-    "github_button": True,
-    # Codecov button
-    "codecov_button": True,
+    # Github star button
+    "github_type": "star",
+    "github_count": "true",
```

Comment:
I like the move to Star instead of Watch.

Bit confused however as to why `github_button` didn't need a count option, but `github_action` does? Does Watch automatically add a count but star does not? If so that makes sense.

Should we use `True` though rather than `true` - abide by our own rules! 😁

I'm not sure about removing the code coverage button. Yes it's not really needed, and yes it's a bit geeky, but this is a project for geeky people and we're very proud of our high code coverage - though I personally want it even higher! See #1314 

```suggestion
    # Github star button
    "github_type": "star",
    "github_count": "True",
    # Codecov button
    "codecov_button": True,
```

Reply:
Cause `github_button` is convenient function of `github_type="watch" and  github_count="true"`

And it's odd that `github_count` only work when we set to `"true"` instead of `True`, maybe it just pass value to https://ghbtns.com and it need `"true"`

Ok, I reset the code coverage button, for geeky 👍

Reply:
OK let's add a comment explaining it explicitly needs to be `"true"` in case anyone thinks this is a mistake.

Reply:
Updated suggested code change.

Reply:
Ok, I'd like to add it